src/nvim/tui/tui.c

```diff
@@ -1437,8 +1450,8 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
     }
   }
 
-  if (!true_xterm) {
-    // Cannot trust terminfo; safer to disable BCE. #7624
+  if (tmux || screen || kitty) {
+    // Disable BCE in some cases we know it is not working. #8806
     unibi_set_bool(ut, unibi_back_color_erase, false);
```

Comment:
👍  Since this drives the `TUIData.bce` value,   `nvim -V3` should correctly reflect whether BCE is in effect.